shell/app/electron_library_main.mm

```diff
@@ -25,6 +26,9 @@ int ElectronMain(int argc, char* argv[]) {
 
 #if BUILDFLAG(ENABLE_RUN_AS_NODE)
 int ElectronInitializeICUandStartNode(int argc, char* argv[]) {
+  if (!electron::fuses::IsRunAsNodeEnabled())
+    return 1;
```

Comment:
`CHECK` perhaps? seems like this function should never be reached if `IsRunAsNodeEnabled` is false.